tribe-ext-ea-additional-options.php

```diff
@@ -0,0 +1,329 @@
+                    'meta_query' => [
+                        'relation' => 'AND',
+                        [
+                            'key' => '_tribe_aggregator_source',
+                            'value' => $source,
+                            'compare' => '='
+                        ],
+                        [
+                            'key' => '_tribe_aggregator_record',
+                            'value' => $postId,
+                            'compare' => '<'
+                        ]
+                    ],
+                ]);
+                foreach ($idsToDelete as $eventId) {
+                    tribe_delete_event($eventId, $deletePermanently);
+                }
+            }
+        }
+
+        /**
+         * Adds extra options to the 'Other URL' settings
+         * 
+         * @param type $options
+         * @return array
+         */
+        public function add_other_url_options($options) {
+            $options[MONTH_IN_SECONDS * 6] = array(
+                'title' => __('Six months', 'tribe-ext-ea-additional-options'),
+                'range' => __('six months', 'tribe-ext-ea-additional-options'),
+            );
+            $options[YEAR_IN_SECONDS] = array(
+                'title' => __('One year', 'tribe-ext-ea-additional-options'),
+                'range' => __('one year', 'tribe-ext-ea-additional-options'),
+            );
+            $options[YEAR_IN_SECONDS * 2] = array(
+                'title' => __('Two years', 'tribe-ext-ea-additional-options'),
+                'range' => __('two years', 'tribe-ext-ea-additional-options'),
+            );
+            return $options;
+        }
+
+        /**
+         * HTML for the additional options for individual imports
+         */
+        public function add_import_options() {
+            $record = new stdClass;
+            if (!empty($_GET['id'])) {
+                $get_record = Tribe__Events__Aggregator__Records::instance()->get_by_post_id((int) $_GET['id']);
+                if (!tribe_is_error($get_record)) {
+                    $record = $get_record;
+                }
+            }
+            $selectedTimezone = get_post_meta($record->post->ID, Tribe__Events__Aggregator__Record__Abstract::$meta_key_prefix . 'timezone', true);
+            $selectedPrefix = get_post_meta($record->post->ID, Tribe__Events__Aggregator__Record__Abstract::$meta_key_prefix . 'prefix', true);
+            $prefixValue = empty($selectedPrefix) ? "" : $selectedPrefix;
+            $tzlist = DateTimeZone::listIdentifiers(DateTimeZone::ALL);
+            $timezoneSelect = '<select name="aggregator[timezone]" id="tribe-ea-field-timezone" class="tribe-ea-field tribe-ea-dropdown tribe-ea-size-large">';
+            $timezoneSelect .= '<option value="">Do not change the timezone.</option>';
+            foreach ($tzlist as $tz) {
+                $timezoneSelect .= '<option value="' . esc_attr($tz) . '"';
```

Comment:
You can use the [selected] function  to avoid an additional conditional

```suggestion
                $timezoneSelect .= '<option value="' . esc_attr($tz) . '"'. selected( $selectedTimezone, $tz, false );
```

[selected]: https://developer.wordpress.org/reference/functions/selected/